src/issue/screens/issue.screen.js

```diff
@@ -40,7 +40,8 @@ const mapStateToProps = state => ({
 });
 
 const mapDispatchToProps = dispatch => ({
-  getIssueCommentsByDispatch: url => dispatch(getIssueComments(url)),
+  getIssueCommentsByDispatch: issueCommentsURL =>
+    dispatch(getIssueComments(issueCommentsURL)),
```

Comment:
Yea, that's why I'm thinking of shortening the line to use `url` instead of `issueCommentsURL` because `prettier` would keep it all on the same line 👍